sdk/appcenter-crashes/src/androidTest/java/com/microsoft/appcenter/crashes/CrashesAndroidTest.java

```diff
@@ -58,6 +59,14 @@
 
     private Channel mChannel;
 
+    /* Filter out the breakpad folder. */
+    FilenameFilter breakpadFilter = new FilenameFilter() {
```

Comment:
If it can be private, it has to.
Also member variables in Android are following `mName` convention except for inner classes sometimes.

Reply:
Done.